app/application.js

```diff
@@ -1,26 +1,33 @@
 "use strict";
 
-var root = 'http://localhost:4000/api';
-var origin = window.location.origin;
+const root = 'http://localhost:4000/api';
+const origin = window.location.origin;
+
+function link(id, text){
+  return `<a href= ${origin}/announcement.html?id=${id}>${text}<br/><br/></a>`
+}
```

Comment:
I think pulling it out of app is fine but I would make this into a function on the `Path` class if possible.